packages/frontend/src/utils/orderByTvl.ts

```diff
@@ -1,20 +1,23 @@
 import { ProjectId, TvlApiResponse } from '@l2beat/shared'
-import { partition } from 'lodash'
 
-export function orderByTvl<T extends { id: ProjectId; isArchived?: boolean }>(
-  projects: T[],
-  tvlApiResponse: Pick<TvlApiResponse, 'projects'>,
-): T[] {
-  const [active, archived] = partition(
-    projects,
-    (project) => !project.isArchived,
+export function orderByTvl<
+  T extends { id: ProjectId; isArchived?: boolean; isUpcoming?: boolean },
+>(projects: T[], tvlApiResponse: Pick<TvlApiResponse, 'projects'>): T[] {
+  const active = projects.filter(
+    (project) => !project.isArchived && !project.isUpcoming,
   )
+  const archived = projects.filter((project) => project.isArchived)
+  const upcoming = projects.filter((project) => project.isUpcoming)
+
   const getTvl = (project: T) =>
     tvlApiResponse.projects[project.id.toString()]?.charts.hourly.data.at(
       -1,
     )?.[1] ?? 0
 
   const sortByTvl = (a: T, b: T) => getTvl(b) - getTvl(a)
 
-  return [...active].sort(sortByTvl).concat([...archived].sort(sortByTvl))
+  return [...active]
+    .sort(sortByTvl)
+    .concat(...upcoming)
```

Comment:
So we don't sort upcoming? Alphabetical order? Or random?

Reply:
I am not sure just for now, lets leave for the next PR